schema.graphql

```diff
@@ -1,15 +1,16 @@
 type Event @entity {
   id: ID!
   blockId: Int!
+  parentBlock: Block!
```

Comment:
Why not change `blockId: Int!` -> `block: Block!` instead? 

That should keep the `blockId` field as is for clients + provide the block connection and shouldn't even need a code change.
 
 Either way the connection should have a test case somewhere.

Reply:
I like idea, but then we would have to again map for each query to match the current o/p (again saying based on current implementation because of the [PR](https://github.com/PolymathNetwork/polymesh-sdk/pull/599/files)). I think originally the idea was to create keep the structure as close as the current one and in doing so it was kept as `blockId: Int!`.

Reply:
When SubQuery sees the relation it generates the id column, and makes the link. We should still have a `blockId` field even after the rename so the schema doesn't change.